pilot/cmd/pilot-agent/main.go

```diff
@@ -60,13 +63,15 @@ const trustworthyJWTPath = "/var/run/secrets/tokens/istio-token"
 // TODO: Move most of this to pkg.
 
 var (
-	role          = &model.Proxy{}
-	proxyIP       string
-	registryID    serviceregistry.ProviderID
-	trustDomain   string
-	pilotIdentity string
-	mixerIdentity string
-	statusPort    uint16
+	role             = &model.Proxy{}
+	proxyIP          string
+	registryID       serviceregistry.ProviderID
+	trustDomain      string
+	pilotIdentity    string
+	mixerIdentity    string
+	statusPort       uint16
+	stsPort          int
+	gCPProjectNumber string
```

Comment:
we already have code in pilot-agent to get this info, do we need it again?

Reply:
Only these two are added: stsPort and gCPProjectNumber, others are format changes.

Reply:
I mean specifically GCP project. Its used for telemetry v2

Reply:
Could you show me which variable is that so I can reuse it?

Reply:
https://github.com/istio/istio/blob/ff1a39d28f5c43bab8139f97505b303dba4eb4ec/pkg/bootstrap/config.go#L105

Reply:
I have revised Pilot-agent main.go to get the project number by API call. Please take a look.

Reply:
Why do hardcoded this vendor specific info?